config/agoradev/develop/alb-redirector.yaml

```diff
@@ -0,0 +1,9 @@
+template:
+  path: alb-redirector.yaml
```

Comment:
I don't see this template in the repo (it was in the original repo https://github.com/Sage-Bionetworks/Agora-infra/blob/develop/templates/alb-redirector.yaml).

Reply:
Thanks for catching that, I've copied over the template now.